tests/ballerina-test/src/test/java/org/ballerinalang/test/types/bytetype/BByteArrayValueTest.java

```diff
@@ -222,4 +222,25 @@ public void testByteArrayLiteral() {
         Assert.assertEquals(returns.length, 1);
         assertResult(bytes, returns[0]);
     }
+
+    @Test(description = "Test return byte array")
+    public void testByteArrayReturn() {
+        byte[] a = ByteArrayUtils.hexStringToByteArray("aaabcfccadafcd341a4bdfabcd8912df");
+        byte[] b = ByteArrayUtils.decodeBase64("aGVsbG8gYmFsbGVyaW5hICEhIQ==");
+        byte[] c = new byte[]{3, 4, 5, 6, 7, 8, 9};
```

Comment:
Instead of a, b, c shall we say val1, val2 and val3 because that makes more sense. Or more meaningful like hexVal, base64Val and numVal.

Reply:
Renamed with better variable names.